From 57bd1141e0719b46a543029c9ff8ed32666a0e49 Mon Sep 17 00:00:00 2001
From: Min RK <benjaminrk@gmail.com>
Date: Thu, 3 Oct 2019 11:09:48 +0200
Subject: [PATCH] Patch SuiteSparse_config.mk

---
 SuiteSparse_config/SuiteSparse_config.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/SuiteSparse_config/SuiteSparse_config.mk b/SuiteSparse_config/SuiteSparse_config.mk
index 19a39032..e5ac60cd 100644
--- a/SuiteSparse_config/SuiteSparse_config.mk
+++ b/SuiteSparse_config/SuiteSparse_config.mk
@@ -370,8 +370,8 @@ SUITESPARSE_VERSION = 5.4.0
         # command line in the Terminal, before doing 'make':
         # xcode-select --install
         CF += -fno-common
-        BLAS = -framework Accelerate
-        LAPACK = -framework Accelerate
+        BLAS ?= -framework Accelerate
+        LAPACK ?= -framework Accelerate
         # OpenMP is not yet supported by default in clang
         CFOPENMP =
     endif
-- 
2.23.0

